docker/root/etc/s6/openssh/setup

```diff
@@ -49,6 +49,7 @@ if [ -d /etc/ssh ]; then
     SSH_DSA_CERT="${SSH_DSA_CERT:+"HostCertificate "}${SSH_DSA_CERT}" \
     SSH_MAX_STARTUPS="${SSH_MAX_STARTUPS:+"MaxStartups "}${SSH_MAX_STARTUPS}" \
     SSH_MAX_SESSIONS="${SSH_MAX_SESSIONS:+"MaxSessions "}${SSH_MAX_SESSIONS}" \
+    SSH_LOG_LEVEL=${SSH_LOG_LEVEL:-"INFO"} \
```

Comment:
```suggestion
    SSH_LOG_LEVEL=${SSH_LOG_LEVEL:+"INFO"} \
```

It should be `:+` here as above for consistency.

Is there any reason to use `:-` here?